### DISCUSSION TITLE
Applying Learnings from Seismic Monitoring to Ocean Acoustic Data

### DISCUSSION THREAD

**Discussion by Artash-N** [softwareunderground/transform-2021-hackathon#21] [Projects]
This year, I have been working on monitoring changes in seismic activity caused by humans during the COVID-19 pandemic. I have been continuously analysing seismic data from the Canadian National Seismograph Network (CNSN) for 13 Canadian Cities.

I am using open-source libraries like ObsPy and SciPy to extract higher frequencies (5 - 20 Hz; Anthropogenic Seismic Vibrations Range vs 0 - 2Hz; Natural Ground Vibrations). To find changes in human activities level, I am using pre-COVID level seismic data as the baseline. I have been producing weekly updates of my analysis. The entire methodology, results of first and and subsequent lockdowns, and weekly updates are available on my WebApp:   http://MonitorMyLockdown.com/ 

There are multiple tools available on the WebApp: to compare weekdays and weekends data, compare seismic vibrations in different frequency ranges, and doing time series analysis. 

TNow, I want to extend my analysis to ocean data. I am exploring two possibilities: either collect and analyse data from underwater seismometers, or use hydrophone data (acoustics). Both of these instruments could possibly shed light on impact of human activities (shipping, sonars, offshore drilling....) on global oceans. I am hoping some of the learnings from terrestrial seismology could be use for analysing ocean data. 

Would welcome collaborations with those who have worked on ocean data.

**Comment by filippo82** [comment]
Hi @Artash-N! welcome back! You should drop a line or 2 about this project idea on the `t21-hackathon` channel on the SWUNG Slack workspace.

**Comment by Artash-N** [comment]
Thanks! I have posted a short description on the channel.
